applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -230,6 +234,10 @@ def LaunchTrainHROM(self, mu_train):
         RedidualsSnapshotsMatrix = []
         for mu in mu_train:
             parameters = self.UpdateProjectParameters(parameters, mu)
+            parameters = self._AddBasisCreationToProjectParameters(parameters)
+            parameters["output_processes"]["rom_output"][0]["Parameters"]["snapshots_interval"].SetDouble(1e30)  # To avoid storing snapshots during training,
+            # we set the value to 1e30. This ensures that the snapshot storage mechanism is not triggered,
```

Comment:
Maybe a stupid question, but which is the reasoning behind adding the ROM output process to then deactivate the snapshot taking?

Reply:
The purpose of adding the ROM output process is to allow the 'calculate_rom_basis_output_process' to pass the ROM output name and folder name as strings.

Reply:
Again, way hacky solution. We need to figure out how to amend this.

Reply:
I will add it to the TODOs list.